lib/msal-common/src/account/IdentityToken.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * IdentityToken claims object with the following signature:
+ * - aud                    - Audience of the token
+ * - iss                    - Token issuer
+ * - iat                    - Issued at
+ * - nbf                    - Not valid before
+ * - exp                    - Expiration time
+ * - amr                    - Subject validation method(s) ["mfa", "rsa", "pwd"]
+ * - email                  - Users' email
+ * - family_name            - Family name
+ * - given_name             - Given name
+ * - idp                    - Identity provider, is the same as the issuer unless it's a guest user.
+ * - ipaddr                 - Users' IP address
+ * - name                   - Mutable username
+ * - oid                    - Immutable object identifier, this ID uniquely identifies the user across applications
+ * - sub                    - Immutable subject identifier, this is a pairwise identifier - it is unique to a particular application ID
+ * - tid                    - The Users' tenant, or '9188040d-6c67-4c5b-b112-36a304b66dad' for personal accounts.
+ * - roles                  - The application roles the user has assigned
+ */
+export type IdentityToken = {
```

Comment:
This is already defined in `TokenClaims` [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-common/src/account/TokenClaims.ts). Not sure if it's exported from msal-common currently, however

Reply:
As discussed with @jasonnutter I was invited to create a PR, see https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/4538#issuecomment-1075465326

This PR would allow access to IdentityToken claims, currently this is exposed as `object` in the **AccountInfo**, with these changes the user has direct access to the identity token claims. as discussed [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/4538#issue-1146825813)

Reply:
Also when checking the [TokenClaims](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-common/src/account/TokenClaims.ts) I see a lot of missing claims and claims that won't be in the token anyway.

Reply:
I'm suggesting to update the existing `TokenClaims` object with your desired behavior rather than creating a duplicate.